RELEASING.md

```diff
@@ -31,6 +32,11 @@ gpg --armor --export-secret-key ${KEY_ID} | pbcopy
 
 While `pbcopy` is macOS specific, similar utilities exist for Linux see [StackExchange](https://superuser.com/a/288333)
 
+## Prepare the release blog post
+
+Prepare the release blog post by opening a PR [kroxylicious.github.io](https://github.com/kroxylicious/kroxylicious.github.io.  Get the PR
+reviewed by your peers, addressing any comments, until the content is agreed.  Don't merge it yet.
```

Comment:
It can merge at any time with a future date in the fronmatter. Not sure that really helps with the problem at hand though.

Reply:
There's always a chance that a release gets delayed and the release date will need to be fixed up. 

If we reach the point where the release is being driven from milestones, the release process could merge the blog posts PRs with the correct milestone as part of the release.  We'd need something to set the release date.  Ideas for later.